hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlStore.java

```diff
@@ -269,9 +269,8 @@ public String toString() {
     }
 
     protected void initTables() {
-        Session session = this.sessions.session();
         for (MysqlTable table : this.tables()) {
-            table.init(session);
+            table.init(this.sessions);
```

Comment:
Please add a config() method to MysqlSessions.Session class instead of passing the sessions object to the table.